src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -633,7 +631,7 @@ def __call__(self):
             if funcName == "repack":
                 self.handleRepackSettings()
 
-            if funcName in ["repack", "merge", "alcaHarvesting" ]:
+            if funcName in ["merge", "alcaHarvesting" ]:
```

Comment:
Dirk, you can actually remove these two lines. I put a generic core override (no matter the task, it will always set numberOfCores) 
https://github.com/hufnagel/WMCore/blob/e8d45081f5995a4934499e09dd8c45ced4bdac52/src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py#L658

and missed this one.